<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>3700</width>
    <height>3300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/ProjectImages/options_2.png</normaloff>:/ProjectImages/options_2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
         </property>
         <property name="text">
          <string>数据库IP  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="settinglineEdit0">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
         </property>
         <property name="text">
          <string>数据库实例</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="settinglineEdit1">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
         </property>
         <property name="text">
          <string>数据库驱动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MS SQLSERVER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MYSQL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>POSTSQL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ORCALE</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="textEdit"/>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
          </property>
          <property name="text">
           <string>数据库名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="settinglineEdit2">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
          </property>
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="settinglineEdit3">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="settinglineEdit4">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;DFKai-SB&quot;;</string>
       </property>
       <property name="text">
        <string>测试连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>settinglineEdit1</zorder>
  <zorder>settinglineEdit2</zorder>
  <zorder>settinglineEdit3</zorder>
  <zorder>settinglineEdit4</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>comboBox</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>textEdit_2</zorder>
  <zorder></zorder>
  <zorder>label_6</zorder>
  <zorder>settinglineEdit0</zorder>
  <zorder>line_4</zorder>
  <zorder>comboBox</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>verticalSpacer_2</zorder>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
